### ISSUE TITLE
Badge images not rendering in webview

### ISSUE THREAD

**Issue by rnystrom** [rnystrom/GitHawk#80]

**Comment by rnystrom** [comment]
e.g. https://github.com/Moya/Moya/issues/1163

**Comment by rnystrom** [comment]
![img_4558](https://user-images.githubusercontent.com/739696/27942485-506d529a-62a5-11e7-8e30-a6388dd08fdf.PNG)

**Comment by Sherlouk** [comment]
Just had a quick look into this, the image url is [this](https://codecov.io/gh/Moya/Moya/pull/762/graphs/tree.svg?token=CjZLHCNgNn&width=650&height=150&src=pr).

Currently in the `ImageCommentImageCell` where we load the image we don't handle if SDWebImage throws us an error? Instead we just make the background clear and get rid of the spinner. We should probably display an error or something?

In this instance the error is: "Downloaded image has 0 pixels". This is because [SDWebImage doesn't support SVG](https://github.com/rs/SDWebImage/issues/1365).

So basically we need an image caching library which supports GIFs, SVGs and anything else is a bonus!

**Comment by rnystrom** [comment]
Ah, awesome. I want to add GIF support and FLAnimatedImage or w/e is already a dependency. Shouldn't be too hard.

**Comment by rnystrom** [comment]
Tracking explicitly in #268
